docs/Server-Methods.md

```diff
@@ -64,6 +65,14 @@ fastify.listen(3000, '127.0.0.1')
   })
 ```
 
+When deploying to a Docker, and potentially other, containers, it is advisable to listen on `0.0.0.0` because they do not default to exposing mapped ports to `127.0.0.1`:
+
+```js
+fastify.listen(3000, '0.0.0.0', (err) => {
+  if (err) throw err
```

Comment:
To be consistent with other examples, can you please replace by:
```
if (err) {
  fastify.log.error(err)
  process.exit(1)
}
```

Reply:
Done.